swagger.yaml

```diff
@@ -183,8 +185,12 @@ paths:
           type: string
         - name: source_metadata
           in: query
-          description: '???'
+          description: >-
+            A JSON blob containing extra source specific information.  This
+            field is free-form, but we generally expect the data will follow
+            one of the formats outlined in the `/schema` folder.
```

Comment:
No, it’ll return a YML file if it’s a `.yml.erb`. All that happens is that the file is first run through the ERB processor when compiling assets, then the `.erb` extension is stripped and it is served exactly as it otherwise would have been. Have tested, too :)